packages/migrations/src/migrations/migrations.ts

```diff
@@ -16,7 +16,16 @@ import {
 
 const sql = sqldb.loadSqlEquiv(import.meta.filename);
 
-export async function init(directories: string | string[], project: string) {
+interface InitOptions {
+  directories: string[];
+  project: string;
+  migrationFilters?: {
+    beforeTimestamp?: string | null;
+    inclusiveBefore?: boolean;
+  };
+}
+
+export async function init({ directories, project, migrationFilters = {} }: InitOptions) {
```

Comment:
This is a breaking change, don't forget your changeset with a `major` version bump.

Reply:
Done in [9874b10](https://github.com/PrairieLearn/PrairieLearn/pull/12789/commits/9874b10585a948c091d808bd346a22c5642ef824).